api/helpers.go

```diff
@@ -159,3 +161,17 @@ func getEntryFromRequest(r *http.Request) (plugin.Entry, string, *errorResponse)
 	entry, err := findEntry(ctx, root, segments)
 	return entry, path, err
 }
+
+func getBoolParam(u *url.URL, key string) (bool, *errorResponse) {
+	vals := u.Query()[key]
+	if len(vals) > 0 {
+		// Take last value
```

Comment:
A `getParam`-type helper would be useful to centralize the "we take the last value of a non-array query param" concept.